src/main/java/com/triviamachine/triviamachine/security/WebSecurityConfig.java

```diff
@@ -0,0 +1,56 @@
+package com.triviamachine.triviamachine.security;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+@EnableWebSecurity
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+    @Autowired
+    UserDetailsService service;
+
+    @Bean
+    public PasswordEncoder getPasswordEncoder() {
+        PasswordEncoder encoder = new BCryptPasswordEncoder();
+        return encoder;
+    }
+
+    @Override
+    protected void configure(AuthenticationManagerBuilder auth) throws Exception {
+        PasswordEncoder encoder = getPasswordEncoder();
+
+        auth.userDetailsService(this.service).passwordEncoder(encoder);
+    }
+
+    @Override
+    protected void configure(HttpSecurity htpp) throws Exception {
+        htpp
+                .cors().disable()
+                .csrf().disable()
+
+                .authorizeRequests()
+                .antMatchers("/admin//*").authenticated()
```

Comment:
extra `/` in there?

It's a bit odd that it's called 'admin' even though it's really any user, but the code simplification was probably a good call :)